lib/Test/Nginx/Socket.pm

```diff
@@ -2686,6 +2711,24 @@ will produce the following line (to C<stderr>) while running this test block:
 You need to remember to set the C<TEST_NGINX_NO_CLEAN> environment to 1 to prevent the nginx
 and other processes from quitting automatically upon test exits.
 
+=head2 response_body_filters
+
+Transform value for the body of the response_body by the a chain of filters.
+
+    --- response_body_filters eval
+    \&CORE::uc
+    --- response_body
+    HELLO
+
+If the response_body_filters is a array, it will be applied a chain of filters one by one for all the requests:
```

Comment:
"it will be applied a chain of ..." do you have any idea ? sorry for my pool english :joy: